<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditGroup</class>
 <widget class="QDialog" name="EditGroup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>298</width>
    <height>117</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Modifier un groupe</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="title">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Modifier un groupe</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="errorLabel">
       <property name="styleSheet">
        <string notr="true">color: red</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="nomLabel">
         <property name="text">
          <string>Nom</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="nomLineEdit">
         <property name="enabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="changeButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Changer</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="deleteButton">
         <property name="text">
          <string>Supprimer</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancelButton">
         <property name="text">
          <string>Annuler</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>EditGroup</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>231</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>234</x>
     <y>109</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>changeButton</sender>
   <signal>clicked()</signal>
   <receiver>EditGroup</receiver>
   <slot>changeGroup()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>75</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>23</x>
     <y>92</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>deleteButton</sender>
   <signal>clicked()</signal>
   <receiver>EditGroup</receiver>
   <slot>deleteGroup()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>176</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>193</x>
     <y>112</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nomLineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>EditGroup</receiver>
   <slot>enableButton(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>74</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>29</x>
     <y>52</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>changeGroup()</slot>
  <slot>deleteGroup()</slot>
  <slot>enableButton(QString)</slot>
 </slots>
</ui>
